ruby/vampires.rb

```diff
@@ -0,0 +1,70 @@
+
+applicant_name = #{name}
+  puts "what is your name"
+  name = gets.chomp
+
+def age_correct
+  puts "what is your age"
+  age = gets.chomp.to_i
+  puts "what year were you born"
+  birth_year = gets.chomp.to_i
+if 2017-birth_year == age
```

Comment:
I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.